packages/mccs/mccs.1.1+5/opam

```diff
@@ -8,10 +8,10 @@ homepage: "http://www.i3s.unice.fr/~cpjm/misc/"
 bug-reports: "https://github.com/AltGr/ocaml-mccs/issues"
 license: "LGPL-2.1 with OCaml linking exception, BSD-3-clause, GPL-3.0"
 dev-repo: "git+https://github.com/AltGr/ocaml-mccs.git"
-build: ["jbuilder" "build" "-p" name]
+build: ["dune" "build" "-p" name]
 depends: [
   "ocaml"
-  "jbuilder" {build & >= "1.0+beta14"}
+  "dune" {build & (>= "1.0+beta14" | < "1.6.0" | > "1.6.1")}
```

Comment:
The `>= "1.0+beta14"` constrain is unnecessary (and probably wrong as `1.0 < 1.0+beta14`)

Reply:
Updated!